server/routes/index.js

```diff
@@ -1,5 +1,5 @@
 import Router from 'koa-router';
-import {index, article, articles, explore, healthcheck, featureFlags, performanceTest, explosion, preview, series} from '../controllers';
+import {index, article, articles, explore, healthcheck, featureFlags, performanceTest, explosion, preview, series, prismicArticle} from '../controllers';
```

Comment:
If someone wants to put this in the lint - then yip.
I personalyl find 1 line easier to read, so long as it's not 100+ characters.